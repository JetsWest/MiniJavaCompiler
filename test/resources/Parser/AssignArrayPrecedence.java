class A {
    public static void main(String[] args) {
        System.out.println(new B().index());
    }

}
class B {
    public int index() {
        int[] a;
        int b;
        a = new int[1];
        a[0] = 5;
        b = a[0];
        return b;
    }
}