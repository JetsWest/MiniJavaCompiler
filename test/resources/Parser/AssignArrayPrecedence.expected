Program
  MainClass A
    Print
      new B().index()
  Class B
    MethodDecl index
      returns int
      int[] a
      int b
      a = new int[1]
      a[0] = 5
      b = a[0]
      Return b